lab-jaren/test/read-files-test.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const expect = require('chai').expect;
+const readFiles = require('../lib/read-files.js');
+
+describe('testing module readFiles', function(){
+  describe('with a file that does not exist', function(){
+    it('should pass an error into the callback', function(done){
+      readFiles([`${__dirname}/../assets/does-not-exist.txt`], function(err, data){
+        //console.log(err);
+        expect(Boolean(err)).to.equal(true);
```

Comment:
Nice job on making the error a Boolean! Duncan usually teaches us to make the syntax```expect(!!err).to.equal(true)``` when testing if an error occurs, but yours also works really well.